container/view.go

```diff
@@ -27,6 +28,24 @@ var (
 	ErrNameNotReserved = errors.New("name is not reserved")
 )
 
+var (
+	// ErrEmptyPrefix is an error returned if the prefix was empty.
+	ErrEmptyPrefix = errors.New("Prefix can't be empty")
+
+	// ErrNotExist is returned when ID or its prefix not found in index.
+	ErrNotExist = errors.New("ID does not exist")
+)
+
+// ErrAmbiguousPrefix is returned if the prefix was ambiguous
+// (multiple ids for the prefix).
+type ErrAmbiguousPrefix struct {
+	prefix string
+}
+
+func (e ErrAmbiguousPrefix) Error() string {
+	return fmt.Sprintf("Multiple IDs found with provided prefix: %s", e.prefix)
```

Comment:
Same (also for a follow-up) should be lowercase, and implement `ErrInvalidParameter` (basically, none of these are a `500` (server) error.